Index: lib/feature4.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/feature4.dart b/lib/feature4.dart
--- a/lib/feature4.dart	(revision 3cd1d90081e079867035b4c6072a52b8f2b343db)
+++ b/lib/feature4.dart	(revision 9138534b9e9aaff46e3d5ab0cf5638c25330127e)
@@ -1,1 +1,3 @@
-//this is feature4 of master branch
\ No newline at end of file
+//this is feature4 of master branch
+
+//this is commit for patch
\ No newline at end of file
