From e8c043a0c56d784dc0f88333c0e5b8b09df88dac Mon Sep 17 00:00:00 2001
From: VidishaLuhar <vidisha.l@simformsolutions.com>
Date: Tue, 21 Feb 2023 10:18:05 +0530
Subject: [PATCH] patch commit

---
 lib/main.dart | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/main.dart b/lib/main.dart
index 008fa38..9e57a27 100644
--- a/lib/main.dart
+++ b/lib/main.dart
@@ -1,5 +1,7 @@
 import 'package:flutter/material.dart';
 
+//patch example
+
 void main() {
   runApp(const MyApp());
 }
-- 
2.39.1

